### ISSUE TITLE
Flight Events not populating in Foreflight

### ISSUE THREAD

**Issue by pilotron1969** [nguyenquyhy/Flight-Events#84]
I'm a bit confused.  I've even diabled firewalls things tmaybe that was it.  Flight Events is saying I'm connected but somehow it's not popluating on my iPad under "Devices"  I've had it working but not consistently.  Could I have some sort of Blocker on?  It's really bizzare.  I'm on the correct network and have even tried putting the IP address in.  The iPad isn't recognizing FE.  Any help would be appreciated.  It's an iPad mini.  Thanks!

**Comment by nguyenquyhy** [comment]
Broadcasting without putting in the IP is usually unreliable due to limits of number of devices to broadcast to in the router. However, I'm surprised even putting in the IP doesn't help.

Do both the PC and the iPad connect to the same Wifi router?

**Comment by pilotron1969** [comment]
Yes.  In FF it provides in "General Info" the SSID and IP Addresses FF is
connected to.  And that's the same WIFI my computer is connected to.  In FE
it says it's "Connected".  However there's no reconizible device to click
onto.

**Comment by pilotron1969** [comment]
I think I figured out what I wasn't doing.  And this has worked a couple times.  When I clicked on "Ident" in FE it populated on my iPad.  I never had to do that before but I'm glad I was able to figure this out.

**Comment by andinogab** [comment]
My issue is current as of V. 2.7.5.0 

How were you able to fix this issue?

**Comment by pilotron1969** [comment]
Yes.  I reset my iPad.  Reinstalled ForeFlight and all is good!  Thanks!!!  RonSent from my iPhoneOn May 13, 2024, at 2:29 PM, andinogab ***@***.***> wrote:﻿
My issue is current as of V. 2.7.5.0
How were you able to fix this issue?
